codice/alliance
# CAL-90 align video with new taxonomy

#### What does this PR do?
Align video input transformer and udp video monitor with new taxonomy, and extract codec names from mpeg-ts data.
#### Who is reviewing it?
@bdeining 
@jrnorth 
@rzwiefel 
@beyelerb 
#### How should this be tested?
Build alliance experimental. Ingest video file or stream. Verify through SOLR that the attributes comply with the new taxonomy.
#### Any background context you want to provide?
Some code was refactored into a new lib. I still need to move some test code over to the new lib. But I wanted to get the basic code up for review. I still need to work on the wiki documentation requirement in the ticket.
#### What are the relevant tickets?
CAL-90
#### Screenshots (if appropriate)
#### Checklist:
